WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -432,11 +435,15 @@ def execute( self ):
     checkCmd = 'dirac-wms-get-wn-parameters -S %s -N %s -Q %s %s' % ( self.pp.site, self.pp.ceName, self.pp.queueName,
                                                                        " ".join( self.cfg ) )
     retCode, result = self.executeAndGetOutput( checkCmd, self.pp.installEnv )
-
-    result = result.split( ' ' )
-    numberOfProcessor = int( result[0] )
-    maxRAM = int( result[1] )
-
+    if retCode:
+      self.log.warn( "Could not get resource parameters [ERROR %d]" % retCode )
```

Comment:
Because, at the beginning I did not want to stop the pilot if we did not have the parameters. 
Fixed now.